app/src/ui/changes/changes-list.tsx

```diff
@@ -981,6 +982,7 @@ export class ChangesList extends React.Component<
               isCommitting: isCommitting,
             }}
             onRowClick={this.props.onRowClick}
+            onRowDoubleClick={this.props.onRowDoubleClick}
```

Comment:
Just curious: why bubbling this up instead of also calling `onOpenItemInExternalEditor`?

Reply:
Ah good catch 🥳 -  no need to hoist this any further 👍🏽